referenceframe/model.go

```diff
@@ -274,3 +275,22 @@ func sortTransforms(unsorted map[string]Frame, parentMap map[string]string, star
 
 	return orderedTransforms, nil
 }
+
+// ModelFromPath returns a Model from a given path.
+func ModelFromPath(modelPath, name string) (Model, error) {
+	var (
+		model Model
+		err   error
+	)
+	switch {
+	case strings.HasSuffix(modelPath, ".urdf"):
+		model, err = ParseURDFFile(modelPath, name)
+	case strings.HasSuffix(modelPath, ".json"):
+		model, err = ParseModelJSONFile(modelPath, name)
+	// case modelPath == "":
+	// 	return model, nil
+	default:
+		return model, errors.New("unsupported kinematic model encoding file passed")
```

Comment:
I don't love using file suffix comparisons like this, since it's just a string and only a mild suggestion at file contents. But ok.

However, if we're going to error on file suffixes, the returned error should be specific about why we're erroring- e.g. "only files with `.json` and `.urdf` file extensions are supported"